installing/installing_aws/installing-aws-outposts-remote-workers.adoc

```diff
@@ -0,0 +1,67 @@
+:_content-type: ASSEMBLY
+[id="installing-aws-default"]
+= Installing a cluster on AWS with remote workers on AWS Outposts
+include::_attributes/common-attributes.adoc[]
+:context: installing-aws-outposts-remote-workers
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Amazon Web Services (AWS) with remote workers machines on AWS Outposts.
```

Comment:
What do you think about changing `remote workers machines on` to `remote workers running on`?